tests/rptest/wasm/topics_result_set.py

```diff
@@ -32,6 +32,18 @@ def __eq__(self, o):
         return self.topic == o.topic and self.partition == o.partition \
             and self.key == o.key and self.value == o.value
 
+    def __hash__(self):
```

Comment:
I'm sure theres a better way to do this. Anyone know?

Reply:
Create a tuple of the elements you want to include in the hash and then it's a one liner: `hash((self.topic, self.partition, self.key, self.value))`

Reply:
Also maybe use tuple for `__eq__`